cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -141,34 +148,44 @@ def _use_sample_sheet_from_housekeeper(self, flow_cell: IlluminaRunDirectoryData
         """
         Copy the sample sheet from Housekeeper to the flow cell directory if it exists and is valid.
         """
-        try:
-            sample_sheet_path: Path = self.hk_api.get_sample_sheet_path(flow_cell.id)
-        except HousekeeperFileMissingError:
-            raise SampleSheetError(
-                f"Sample sheet for flow cell {flow_cell.id} does not exist in Housekeeper"
-            )
```

Comment:
Are we ok with removing this `try-except`?

Reply:
yes, the purpose of this one was to translate the error from a HousekeeperFileMissingError to a SampleSheetError. All CgErrors are caught later